services/web/server/src/simcore_service_webserver/projects/_projects_service.py

```diff
@@ -1565,6 +1565,17 @@ async def try_open_project_for_user(
         )
         async def _open_project() -> bool:
             with managed_resource(user_id, client_session_id, app) as user_session:
+                # check if the project is already opened
+                if (
+                    current_project_ids := await user_session.find(PROJECT_ID_KEY)
+                ) and current_project_ids == [f"{project_uuid}"]:
```

Comment:
Q: Should this check not be? This might fail if a different project is already opened
```suggestion
                ) and f"{project_uuid}" in current_project_ids:
```

Reply:
nevertheless will change to your option, in case that changes.